app/cafe.py

```diff
@@ -0,0 +1,27 @@
+import datetime
+from app.errors import NotVaccinatedError
+from app.errors import OutdatedVaccineError
+from app.errors import NotWearingMaskError
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Person is not vaccinated!")
+
+        elif visitor["vaccine"].get(
+                "expiration_date"
+        ) is not None and visitor[
+            "vaccine"
+        ][
+            "expiration_date"
+        ] < datetime.date.today():

```

Comment:
As for me, it is so bad written. And so hard to read this code....
```suggestion
        expiration = visitor["vaccine"].get("expiration_date")
        if expiration is not None and expiration < datetime.date.today():
```